airgun/entities/virtwho_configure.py

```diff
@@ -27,6 +27,25 @@ def _reset_values(self, values):
             vals.update({'hypervisor_type': mapping[hypervisor_value]})
         return vals
 
+    def permissions(self):
+        """Assert if the config can be viewed/created"""
```

Comment:
we can use 'check' instead Assert. I think we use Assert more like in the testing phase.

Reply:
Absolutely